tutorials/Tutorial_Gallery.ipynb

```diff
@@ -57,7 +57,8 @@
     "- [Quantile Loss](./Quantile_Loss.ipynb)\n",
     "- [Murphy Diagrams](./Murphy_Diagrams.ipynb)\n",
     "- [Flip-Flop Index](./Flip_Flop_Index.ipynb)\n",
-    "- [Consistent Scores](./Consistent_Scores.ipynb)"
+    "- [Consistent Scores](./Consistent_Scores.ipynb)\n",
+    "- [Threshold Weighted Scores](./Threshold_Weights_scores.ipynb)"
```

Comment:
I suggest renaming the notebook file to "Threshold_Weighted_Scores.ipynb" for consistency. There are some links which point to that (e.g. the consistency notebook and included.md) which don't resolve as a result.